### DISCUSSION TITLE
Node ID's

### DISCUSSION THREAD

**Discussion by ActualHamish** [geode-sdk/sdk#815] [General]
Can't get into geode BC of node IDs, prompts me to update it by clicking on this MASSIVE logo on the screen but there's no update available, 2.206 bug :p. My question is why does this happen?

**Comment by Cvolton** [comment]
disable or uninstall click sounds mod

**Reply by ActualHamish** [reply]
Thank you! I'll assume that the individual mods will fix during the passage of time :)
